sw/device/rom_exts/manifest.md

```diff
@@ -184,13 +191,15 @@ Notes:
     This is used when signing and validating the image. This happens in the
     Mask ROM, as well as during firmware update.
 
-1.  **Usage Constraints** This is a 32-bit enumeration value which denotes what
-    kinds of devices the ROM_EXT image may be used on.
+1.  **Usage Constraint** This is a 256-bit unsigned numeric value.
 
-    This is used by the Mask ROM to validate the image is running on the right
-    kind of hardware.
+    This allows the ROM_EXT author to constrain a ROM_EXT to a single device or
+    a group of devices, such that the ROM_EXT will not validate (and therefore
+    will not boot) on devices not in that group.
 
-    **Open Q** What possible values should this have?
+    The Mask ROM will use this value to calculate a 8192-byte "device usage
```

Comment:
1024-byte value

Reply:
Woops, got my bytes and bits all mixed up.